openedx/adg/admin_dashboard/admin_settings/tests/test_views.py

```diff
@@ -13,39 +13,70 @@
 from openedx.core.djangolib.testing.utils import skip_unless_lms
 
 
+@pytest.fixture(name='admin_accounts_list_request')
+@skip_unless_lms
+def admin_account_list_view_request(adg_admin):
+    """
+    Fixture which returns ADG admin user.
+    """
+    url = reverse('admin_accounts')
+    admin_accounts_get_request = RequestFactory().get(url)
+    admin_accounts_get_request.user = adg_admin
+
+    return admin_accounts_get_request
+
+
+@pytest.fixture(name='pending_invitations_list_request')
+@skip_unless_lms
+def pending_invitations_view_list_request(adg_admin):
+    """
+    Fixture which returns ADG admin user.
+    """
+    url = reverse('pending_invitations')
+    pending_invitations_get_request = RequestFactory().get(url)
+    pending_invitations_get_request.user = adg_admin
+
+    return pending_invitations_get_request
+
+
 @pytest.mark.parametrize('total_admins, expected_count', [
-    (5, 10),
-    (10, 20),
+    (5, 11),
+    (10, 21),
 ])
 @pytest.mark.django_db
 @skip_unless_lms
-def test_admin_accounts_list_queryset(total_admins, expected_count):
+def test_admin_accounts_list_queryset(total_admins, expected_count, admin_accounts_list_request):
     """
-    Test queryset for `AdminAccountListView` returns expected number of active admins.
+    Test queryset for `AdminAccountListView` returns expected number of active admins. Number of accounts will be
+    [(`total_admins` * 2) + 1], in which (`total_admins` * 2) will be created in this test and 1 is the adg_admin
+    user attached with the request.
     """
     create_admin_accounts(total_admins, ADG_ADMIN, True)
     create_admin_accounts(total_admins, BU_ADMIN, True)
     create_admin_accounts(total_admins, BU_ADMIN, False)
-    assert AdminAccountListView().get_queryset().count() == expected_count
+
+    list_view = AdminAccountListView()
+    list_view.request = admin_accounts_list_request
+
+    assert list_view.get_queryset().count() == expected_count
```

Comment:
Are we testing this for both BU admins and ADG admins?

Reply:
No, But we should, Will add tests for that.

Reply:
That was my point as well. Thank you!